osquery/tables/system/posix/tests/yum_sources_tests.cpp

```diff
@@ -0,0 +1,72 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include<sstream>
+
+#include <gtest/gtest.h>
+
+#include <osquery/sql.h>
+
+namespace osquery {
+namespace tables {
+
+void parseYumConf(std::istream&, QueryData&ults, std::string&epos_dir);
+
+class YumSourcesTests : public testing::Test {};
+
+TEST_F(YumSourcesTests, parse_yum_conf) {
+  QueryData results;
+  std::string repos_dir;
+
+  std::istringstream stream1("");
+  parseYumConf(stream1, results, repos_dir);
+  ASSERT_EQ(repos_dir, "/etc/yum.repos.d");
+  ASSERT_EQ(results.size(), (unsigned long) 0);
+
+  std::istringstream stream2(R"STRLIT(
+# Some comment
+
+[main]
+cachedir=/var/cache/yum
+reposdir=/etc/local/yum.repos.d
```

Comment:
Be explicit that this is supposed to not exist. Or am I getting wrong what you're doing here?

Reply:
Yeah it's not supposed to exist